core/prettier-plugin-handlebars/README.md

```
@@ -0,0 +1,47 @@
+[![npm downloads](https://img.shields.io/npm/dw/@stephansama/prettier-plugin-handlebars?labelColor=211F1F)](https://www.npmx.dev/package/@stephansama/prettier-plugin-handlebars)
+
+[![prettier](https://img.shields.io/badge/prettier-3.8.1-F7B93E.svg?logo=prettier&logoColor=ffffff&labelColor=F7B93E)](https://npmx.dev/package/prettier)
+[![tsdown](https://img.shields.io/badge/tsdown-0.21.10-3178C6.svg?logo=rolldown&logoColor=ffffff&labelColor=3178C6)](https://npmx.dev/package/tsdown)
+
+<!-- BADGE end -->
+
+</div>
+
+Prettier plugin that automatically assigns the default parser for various handlebars files
+
+##### Table of contents
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h5

(MD001, heading-increment)